clang/test/dpct/different_version_warning/output_ref.txt

```diff
@@ -1 +1,2 @@
-// CHECK: Warning: Incremental migration requires the same version of dpct. Migration continues with incremental migration disabled. See https://software.intel.com/content/www/us/en/develop/documentation/intel-dpcpp-compatibility-tool-user-guide/top.html for more details.
\ No newline at end of file
+// CHECK: [Warning] The file '/home/testfolder/test.cu' has been changed during incremental migration.
+// CHECK: Warning: Failed to load {{(.+)}}.yaml. Migration continues with incremental migration disabled. See https://software.intel.com/content/www/us/en/develop/documentation/intel-dpcpp-compatibility-tool-user-guide/top.html for more details.
```

Comment:
Because when cuda file's hash is checked failed, the yaml file will be ignored, so the version of dpct will not be checked.